### DISCUSSION TITLE
Feedback page keeps showing with new tab.

### DISCUSSION THREAD

**Discussion by allanswer** [Able-Bear-Software/tab-nodes-tree#2] [General]
Hi, I just subscribed pro and everything looks smooth. But I found the feedback page will pop up sometimes. 
<img width="950" alt="image" src="https://github.com/user-attachments/assets/520fb581-4258-4483-894a-3661ff05e3ea">
Is there any way to make sure this page just show one time and won't pop up in the future ?
Thanks !

**Comment by zeroarst** [comment]
Thanks for reporting this. This page is shown upon first time installation and each update. I will think of a better solution for it.

**Comment by zeroarst** [comment]
The page will be amended slightly to show the current extension version and include a link to the latest changes. Target version: 1.2.0.
